test/lib/css_coverage_test.rb

```diff
@@ -29,7 +29,7 @@ def setup
     # Cleanup the data to make sure it's only one selector per item
     @css_data = @css_data
                 .flat_map { |c| c.gsub(/(\w)\./, '\1 .').split(/[\s:\[+>]+/) }
-                .select { |c| c.starts_with?(".") }
+                .select { |c| c.start_with?(".") }
```

Comment:
Huh how did this ever work??

Reply:
I *think* ActiveSupport in Rails will alias these, however, I think `start_with?` is the generic String method that works everywhere, and I think that Sorbet generally prefers we just use the `start_with?` method on String itself rather than ActiveSupport's aliasing.